clients/web/apps/rook-dashboard/src/style.css

```
@@ -0,0 +1,316 @@
+@import url("@corvus/shared/base.css");
```

Stylelint (17.7.0)

[error] Expected "url("@corvus/shared/base.css")" to be ""@corvus/shared/base.css"" (import-notation)

(import-notation)